README.md

```diff
@@ -1,3 +1,39 @@
 # SnowMan
 
-This is a console Swift application. It plays like Hangman but without the gore.
+Snowman is a Swift console application. The object of the game is to guess the given word or phrase before the snowman "melts". With each incorrect guess, the snowman will lose one of its snowman features until he is but a puddle. 
+
+## Requirements
+
+[Swift](https://swift.org/) 
+[Swift Package Manager](https://swift.org/package-manager/)
```

Comment:
Probably want these to be:
```Markdown
* [Swift](https://swift.org/) 
* [Swift Package Manager](https://swift.org/package-manager/)
```

otherwise they show up on the same line